### ISSUE TITLE
Select default signup page using loginWithRedirect method

### ISSUE THREAD

**Issue by Adnan-Hussain22** [auth0-samples/auth0-react-samples#196]
Im using loginWithRedirect method to authenticate the user if not authenticated. Basically this will redirect me to the login page. But the problem is that in certain conditions i want the user to be redirected directly to signup instead of login.
I have check loginWithPopup method, it provides you with `screen_hint` option to provide a default selected page (signup | login).
How can i do it with loginWithRedirect ?
Basically loginWithRedirect, redirects you to /authorize. 
I have checked the auth0 docs of new universal login, in signup section it is listed that while redirecting to /authorize you can pass screen_hint=signup as parameter. 
But how can i do that with loginWithRedirect method ?

Using,
New universal login,
auth0-spa-js: ^1.6.4

**Comment by stevehobbsdev** [comment]
Hi @Adnan-Hussain22, the `loginWithRedirect` method should still allow you to pass through custom parameters such as the `screen_hint` parameter. Have you tried this so far?

Let us know how you get on.

**Comment by Adnan-Hussain22** [comment]
Yeh, its working.
@stevehobbsdev thanx. But it is not in the docs, and also not provided in `RedirectLoginOptions`. I guess it should be updated so that no one is getting this problem in future.

BTW thanx alot and also im closing this issues as it is working
